### PR TITLE
Script for defect 2395

### PR DESCRIPTION
**Pull request by TMyza**
ATF Test Scripts to check [#2395](https://github.com/smartdevicelink/sdl_core/issues/2395)

This PR is **[ready]** for review.

### Summary
Script with check issue

### ATF version
develop

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/Defects/5_0/2395_SDL_must_store_OnWayPointChange_internally_and_resend_this_notification_to_each_subscribed_apps.lua [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+                hmi_levels = {"BACKGROUND", "FULL", "LIMITED"}
+            },
+            OnWayPointChange =  {
+                hmi_levels = {"BACKGROUND", "FULL", "LIMITED"}
+            }
+        }
+    }
+    tbl.policy_table.functional_groupings["NewTestCaseGroup"] = OWgroup
+    tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.appID].groups = {"Base-4", "NewTestCaseGroup"}
+    tbl.policy_table.app_policies[config.application2.registerAppInterfaceParams.appID] = tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.appID]
```
Comment (aderiabin) [inline test_scripts/Defects/5_0/2395_SDL_must_store_OnWayPointChange_internally_and_resend_this_notification_to_each_subscribed_apps.lua]: @TMyza Please avoid long sequences of table field data receiving in simple cases. 
There is proposed solution here:
```
local testGroups = { "Base-4", "NewTestCaseGroup" }
tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.appID].groups = testGroups
tbl.policy_table.app_policies[config.application2.registerAppInterfaceParams.appID].groups = testGroups
tbl.policy_table.app_policies[config.application3.registerAppInterfaceParams.appID].groups = testGroups
```

**Comment by dboltovskyi** [comment]
Livio team please do not review this PR for now since it hasn't passed our internal review yet.
Review and appropriate updates going be done till 1st of November.
Thank you.

**Comment by GetmanetsIrina** [comment]
PR is rebased on current develop(892f229) branch.
Script is applicable for all policy flows and transport types.
Script has been moved to 6_1 folder.
Test set for script is added.

Script is updated according to [sequence diagram from issue](https://user-images.githubusercontent.com/11210973/81415096-a16e6800-9150-11ea-9409-9d12c6688144.png).

**Comment by iCollin** [comment]
@GetmanetsIrina Thank you for rebasing script.

I notice that script has incorrect expectation within `subscribeWayPointsAlreadySubscribed2apps`, it should be expecting `notifParamsUpd`.
Additionally, I think the parameters should be removed from the apps 1 and 2 expectations in this method since we need to expect no `OnWayPointChange` notification at all.

As I am merging fix for issue, when will this PR be ready for review?

**Comment by dboltovskyi** [comment]
@iCollin Thank you for the review. Please find updates in 3f20c6482fcd652ef010edc0ab81bc595a09273d. 
In addition script was moved to `6_2` folder

**Pull request merged**